lib/htmlValidator.js

```diff
@@ -38,9 +38,6 @@ module.exports = function (app, callback) {
     validatorOptions.port = params.htmlValidator.port
```

Comment:
Will this trump the node ENV?

Reply:
I know source configs trumps whatever the validator is set to with the Node ENV but we aren't checking the Node ENV here

Reply:
[https://github.com/rooseveltframework/roosevelt/blob/master/lib/sourceParams.js#L125-L149](HTML Validtor source configs) is trumped finally at the end of source configs by [this](https://github.com/rooseveltframework/roosevelt/blob/master/lib/sourceParams.js#L155-L157).

Reply:
Yeah sorry I was reading this wrong, nevermind. Looks fine